src/TodoList.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+    <form>
+        <input type="text" v-model="todo">
+        <button @click.prevent="addToDo">Submit</button>
+        <button @click="todos= []">Clear</button>
```

Comment:
Create a clearTodos function inside script and call this function on click event